.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -9,6 +9,15 @@ labels: kind/bug
 
 **What you expected to happen**:
 
+<!-- Please select an area, component, kind, platform and priority for this issue. This helps the community categorizing it. -->
+<!-- Replace below TODOs or exchange the existing identifiers with those that fit best in your opinion. -->
+**How to categorize this issue?**
+/area/TODO <!-- audit-logging|auto-scaling|backup|certification|control-plane-migration|control-plane|cost|delivery|dev-productivity|disaster-recovery|documentation|high-availability|logging|metering|monitoring|networking|open-source|operations|ops-productivity|os|performance|quality|robustness|scalability|security|storage|testing|usability|user-management -->
```

Comment:
Can you add `space space` at the end of each line to force a line break?
Currently, all `/commands` are in the same line behind the header, which is not intended, I assume.

Reply:
I think this will handled automatically by GitHub - they have their own markdown version and a line break is indeed a line break, so when opening a PR/issue it should work.

Reply:
Interesting, I clicked "View File" and they display it in one line there...

Reply:
AFAIK: In this view they render it as standard markdown file. But in GitHub issues/pull request comments they
respect
the
newline.

See? :)

Reply:
Ok, that's confusing :) 
But I guess, then it's fine.